lms/static/scripts/frontend_apps/components/LMSFilePicker.js

```diff
@@ -233,8 +220,21 @@ export default function LMSFilePicker({
         />
       );
       break;
+    case 'reload':
+      continueButton = (
+        <Button
+          onClick={() => {
```

Comment:
This anonymous function here has enough logic that it seems like extracting it to a named function in the component may be ideal...